server/server/migrations/migrations/add_child_range_collection_027.py

```diff
@@ -0,0 +1,13 @@
+from common.arangodb import get_db
+from migrations.base import Migration
+
+
+class SecondMigration(Migration):
+    migration_id = 'add_child_range_collection_027'
+    tasks = ['create_collections']
+
+    def create_collections(self):
+        db = get_db()
+
```

Comment:
The same here, I guess you can remove it.

Reply:
Well, yes, you can do it without skipping a line, although it is only used to visually separate logic
but I also don't think it's a big deal to reopen the pull-request for that 
you can also look at other migration files and see that "annoying" line skip there =D